### DISCUSSION TITLE
How to set margin with typst?

### DISCUSSION THREAD

**Discussion by dgkeyes** [quarto-dev/quarto-cli#7372] [Q&A] [answered]
### Description

I'm trying to set the margins in a Typst format PDF document created in Quarto. Here's my attempt to set a top margin of 5 inches:

````qmd
---
title: "My Document"
format: 
  typst:
    papersize: us-letter
    fontsize: "20pt"
    top: "5in"
---

Nisi ipsum reprehenderit Lorem commodo excepteur fugiat laborum incididunt. Reprehenderit nulla sunt consectetur excepteur veniam aute in officia. Eu nulla consequat proident minim esse excepteur sit tempor cillum occaecat. Fugiat commodo aliqua laboris labore magna labore id eiusmod veniam officia consequat fugiat. Cupidatat culpa laborum ad ad. Pariatur magna commodo dolore pariatur nisi eu ullamco in commodo.
````

The `top: "5in"` line doesn't seem to actually change anything. Am I doing something wrong?

**Comment by mcanouil** [comment, marked as answer]
`top` can't work because it is not a valid option, see <https://quarto.org/docs/prerelease/1.4/typst.html#typst-format>.

<img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/5893090c-d2dd-488a-9be1-3cdb43447538">

**Reply by dgkeyes** [reply]
Ah, perfect! I didn't realize margin needed to be separate like that.
